packages/foreman/rubygem-sprockets/rubygem-sprockets.spec

```diff
@@ -3,10 +3,11 @@
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name sprockets
+%global gem_require_name %{gem_name}
```

Comment:
What's this new macro?

Reply:
Appears that you added it about two years ago (https://github.com/theforeman/foreman-packaging/pull/4921/files#diff-6192aa78a351560738c958b59214bab5d795aaf4c53870c0e9a56446a92a833dR8) and that's how long it has been since this gem had a spec update.